index.js

```diff
@@ -132,7 +123,10 @@ class Provider {
 
     const uniqueId = this.config.uniqueId
     return this.fetch()
-      .then(({ results }) => this._randomArray(results))
+      .then(({ results }) => {
+        const random = Math.floor(Math.random() * results.length)
+        return results[random]
```

Comment:
`randomArray` was only used once in the `random` method. So unless you also want to use it elsewhere I thought it was best to just implement it in the `random` method.